src/App.tsx

```diff
@@ -1,24 +1,208 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { Filter, Todo } from './types/Todo';
+import * as todosService from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { TempTodo } from './components/TempTodo';
 
-const USER_ID = 0;
+const USER_ID = 11641;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [todoTitle, setTodoTitle] = useState('');
+  const [filterBy, setFilterBy] = useState<Filter>(Filter.all);
+  const [hasCompleted, setHasCompleted] = useState(false);
+  const [quantity, setQuantity] = useState(0);
+  const [hideNotification, setHideNotification] = useState(true);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isFocused, setIsFocused] = useState(true);
+
+  useEffect(() => {
+    const filteredTodos = todos.filter((todo) => todo.completed);
+
+    setQuantity(todos.length - filteredTodos.length);
+  }, [todos]);
+
+  useEffect(() => {
+    setErrorMessage('');
+    todosService
+      .getTodos(USER_ID)
+      .then((data) => {
+        setTodos(data);
+      })
```

Comment:
```suggestion
      .then(setTodos})
```
this can be shortened